index.html

```diff
@@ -0,0 +1,239 @@
+      </div>
+
+    </div>
+    <div class="about-me">
+      <section class="about-me-section">
+        <h2>About me</h2>
+        <p class="about-me-text">
+          I am a skilled Full-Stack Web Developer who loves making pages with
+          beautiful UI. I have studied at Microverse, and I am experienced in
+          remote work.
+          <br />
+          I've also been a good helping hand to a lot of developers in
+          different communities!
+          <br />
+          if you'd like to get in contact about potential job opportunities,
+          please reach out!
+        </p>
+        <a href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+          class="link" target="_blank" rel="noreferrer noopener">
+          Get my resume
+        </a>
+        <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+      </section>
+      <hr />
+      <div class="about-me-section stacks">
+        <ul>
+          <li class="languages list-title">
+            Languages
+            <ul>
+              <li>JavaScript</li>
+              <li>TypeScript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>Sass</li>
+            </ul>
+          </li>
+        </ul>
+        <ul>
+          <li class="frameworks list-title">
+            Frameworks
+            <ul>
+              <li>React & Redux</li>
+              <li>Material-UI</li>
+              <li>Jest</li>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+            </ul>
+          </li>
+        </ul>
+        <ul>
+          <li class="skills list-title">
+            Skills
+            <ul>
+              <li>GitHub</li>
+              <li>CodePen</li>
+              <li>GitLab</li>
+              <li>Terminal</li>
+              <li>WSL</li>
+              <li>Remote Pair-Programming</li>
+              <li>Teamwork</li>
+              <li>Mentoring</li>
+              <li>Code Reviews</li>
+              <li>Mobile/Responsive Development</li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </main>
+
+  <footer class="contact-form">
+    <div class="contact-form-header">
+      <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+    </div>
+    <form action="https://formspree.io/f/mbjqyrqj" method="POST">
+      <div class="form-group">
+        <input type="text" name="name" id="fullname" maxlength="30" placeholder="Write your full name" required />
+        <input type='email' name="email" id="email" placeholder="Write your email" required />
+        <textarea class="form-group-textarea" name="tarea" maxlength="500" placeholder="Enter text here"
+          required></textarea>
+      </div>
+      <button id="sub-btn" type="submit" class="btn-1">Get in touch</button>
+    </form>
+  </footer>
```

Comment:
- [ ] Just as it was mentioned in the [last review](https://github.com/sumairq/cr-training-contact-form/pull/2#discussion_r906632661), you should ensure that your design matches the template on [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). This is a requirement.